### DISCUSSION TITLE
15 settembre 2022 - esercizio 1

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#17] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/276882b5-7193-4505-88cc-9566516035de)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/d242a4a1-c2bd-462e-9de1-efa59913b444)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/121675ea-e849-4bb4-97cd-ae6b588913f6)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/ec89b92d-2f89-499f-a006-87d74db242a0)

**Comment by niccolo-fato** [comment]
![IMAGE 2024-01-14 11:28:05](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/9780162b-a526-4358-8ae4-e9fc8c8040e9)

**Comment by Elia-Belli** [comment]
**(1)**
- Voli diretti a NewYork: $Voli\\_ NY= Volo\bowtie_{(Arrivo=NY)}(\rho_{(NY\leftarrow Sigla)}(\sigma_{(Città\ =\ NewYork)}Aeroporto))$
- Viaggi tra il 2019 e il 2021: $Viaggi\\_ 1921= \sigma_{(01-01-2019\leq Data\lt 01-01-2022)}Viaggio$
- Voli per NY tra il 2019 e il 2021: $Voli\\_ NY\\_ 1921=Voli\\_ NY\bowtie_{(Sigla=SiglaVolo)}Viaggi\\_ 1921$
- Aerei costruiti prima del 2010 e mai revisionati: $Aerei= \sigma_{(AnnoC\lt 2010\ \land\ AnnoR\ =\ 0000)}Aereo$
- Risposta: $\pi_{(Modello)}(Aerei\bowtie_{(ID=Aereo)}Voli\\_ NY\\_ 1921)$

**(2)**
- Voli che partono da Roma dopo le 23:00 : $Voli\\_ RM=(\sigma_{(Orario\gt 23:00)}Volo)\bowtie_{(Partenza\ =\ RM)}(\rho_{(RM\leftarrow Sigla)}(\sigma_{(Città\ =\ Roma)}Aeroporto))$
- Viaggi partiti da Roma dopo le 23:00 : $Viaggi\\_ RM=\pi_{(ID)}(Viaggio\bowtie_{(Sigla=SiglaVolo)}Voli\\_ RM)$
- Personale dei viaggi da Roma dopo le 23:00: $Pers\\_ RM=\pi_{(CF,Nome,Cognome)}((Viaggi\\_ RM\bowtie_{(ID=Viaggio)}Equipaggio)\bowtie_{(Pers=PID)}(\rho_{(PID\leftarrow ID)}Personale))$
- Risposta: $(\pi_{(CF,Nome,Cognome)}Personale) - Pers\\_ RM $
